packages/db/src/lib/timestamp.ts

```diff
@@ -0,0 +1 @@
+export const translateDateToUTC = (date: Date): number => date.getTime() / 1000;
```

Comment:
I get a little nervous maintaining our own Date lib related functionality, but this seems safe enough relative to using a much heavier package